BasicAuth.py

```diff
@@ -1,13 +0,0 @@
-## Class to be used as auth mechanism for the requests module

```

Comment:
Please move the file with "git mv" before making change to it to retain the history and ease the review process (as you did for BearerAuth.py).

Reply:
Sorry will fix this. Thought I did.